### ISSUE TITLE
registration form: captcha and policy-links not accessible in safari

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3544]
**URL:** https://mein.berlin.de/accounts/signup/?next=/
**user:** any
**expected behaviour:** accessible registration form
**behaviour:** tab jumps from the last checkbox right to register button, skipping the captcha and also the links to user and data protection policy. 
**important screensize:**
**device & browser:** safari only (chrome, firefox and edge are fine)
**Comment/Question:**

**Comment by phillimorland** [comment]
Also label on other browsers not tabbed like normal label, treated as website content therefore making it difficult to complete

**Comment by phillimorland** [comment]
@CarolingerSeilchenspringer can you add mac OS version and safari version? there seeme to be differences between older and newer, it's good to know if this affects both or not

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland sure, I tested on macOS Big sur, Safari 14.0.3

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland I will also add this to the milestone as a11y-enhancement-opportunity :-)

**Comment by philli-m** [comment]
@jostertag can you reproduce this? tried on browser stack but only available safari version is 14.1 and my current setup Ventura 13.4.1 and safari 16.5.1 seem to work, I will look into the focus ring however

**Comment by jostertag** [comment]
@philli-m I tried it with ventura 13.1. and it worked as well. I don't know how we could test it with older versions of safari.

**Comment by philli-m** [comment]
@jostertag I have made a bunch of a11y improvements that should help
